gc/gc-iceberg-inttest/src/intTest/java/org/projectnessie/gc/iceberg/inttest/AbstractITSparkIcebergNessieObjectStorage.java

```diff
@@ -153,8 +153,10 @@ static Stream<TestCase> testCases() {
             .namespace("tc_3")
             .addSteps(
                 expiredDdl(
-                    "CREATE TABLE nessie.tc_3.tbl_a (`id\"` int, name string) "
-                        + "PARTITIONED BY (`id\"`)"))
+                    // Note: intentional special chars in the column name
+                    // TODO: debug why `#` breaks this test
```

Comment:
URI fragment?

Reply:
Also, let's have special characters in table and namespace names.
I'm okay to have those in a follow-up.

Reply:
Apparently... but I'd like to check whether AWS chokes on it or MinIO or Iceberg... will probably have to do that tomorrow.